### ISSUE TITLE
Bug 58069601: [React Native Gallery -> TouchableWithoutFeedback]: 'TouchableWithoutFeedback' menu item present in left navigation pane is getting truncated when text scaling is set 200%.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#610]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0

AT: Text Scaling  
Resolution: 1280*768

Pre-Requisite:
Text Scaling (Settings->Accessibility->Text size-> Drag the slider to 200% ->Apply))

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Legacy' button present in left navigation pane and expand it.
Now observe 'TouchableWithoutFeedback' after applying text scaling.
Actual Result:
'TouchableWithoutFeedback' menu item present in left navigation pane is getting truncated when text scaling is set 200%.

Expected Result:
'TouchableWithoutFeedback' menu item present in left navigation pane should be visible clearly to the user without any data loss and truncation.

User impact:

Users with low vision will not be able to consume the content of the button if it is not visible in Text scaling mode.

MAS Reference:
[MAS 1.4.4 – Resize Text](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b5BE8E205-7508-414C-A273-657770321DA0%7d&file=MAS%201.4.4%20%E2%80%93%20Resize%20Text.docx&wdLOR=cD434D0F2-F515-4FD8-B4DA-BA99282968ED&action=default&mobileredirect=true) 

Trap ID:
[7.1 - Data Loss](https://aka.ms/Data_Loss)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
Applicable for fabric.
